conditions/README.md

```diff
@@ -0,0 +1,341 @@
+| category[0].coding[0].display                          | fixed value = `Problem List Item`                                                                                                                                                                  |
+| code                                                   | mapped from `ObservationSatement / code` where the `Observationstatement` is referenced in `LinkSet / component / statementRef / id [@root]`                                                       |
+| subject.reference                                      | reference to the mapped [Patient](../patient/README.md)                                                                                                                                            |
+| context.reference                                      | reference to the associated [Encounter](../encounters/README.md)                                                                                                                                   |
+| onsetDateTime                                          | `LinkSet / effectiveTime / low [@value]` or else `LinkSet / effectiveTime / center [@value]` or else `LinkSet / availabilitytime [@value]`                                                         |
+| abatementDateTime                                      | `LinkSet / effectiveTime / high [@value]`                                                                                                                                                          |
+| assertedDate                                           | `ehrComposition / author / time [@value]` or else `EhrExtract / availabilityTime [@value]`                                                                                                         |
+| asserter.reference                                     | [Practitioner](../practioners/README.md) referenced by `ehrComposition / participant2[0] / agentRef / id [@root]`                                                                                  |
+| note.text[index]                                       | fixed value = `"Defaulted status to active : Unknown status at source"` if `linkSet / code [@code]` does not equal `394774009` (active) or `394775005` (inactive)                                  |
+| note.text[index]                                       | fixed value = `"Unspecified Significance: Defaulted to Minor"` if `LinkSet / code / qualifier / name [@code]` does not equal `386134007`                                                           |
+| note.text[index]                                       | `ObservationStatement / pertinentInformation / pertinentAnnotation / text` where the `observationStatement` is referenced by `LinkSet / conditionNamed / namedStatementRef / id [@root]`           |
```

Comment:
Can we be clear that `ObservationStatement / pertinentInformation` is a List , and that a note is added for each?

Reply:
Also, looking at this code, there appears to be an additional note generated when the `LinkSet / code / originalText` is populated?

Reply:
I'll move notes to a subsection so this can be explained more clearly. And add the `originalText` mapping